lxd/storage/drivers/driver_zfs_patches.go

```diff
@@ -19,3 +23,61 @@ func (d *zfs) patchStorageCreateVM() error {
 
 	return nil
 }
+
+func (d *zfs) patchStorageZFSMount() error {
+	datasets, err := d.getDatasets(d.config["zfs.pool_name"])
+	if err != nil {
+		return err
+	}
+
+	for _, dataset := range datasets {
+		// Skip snapshots.
+		if strings.Contains(dataset, "@") {
+			continue
+		}
+
```

Comment:
You should be skipping bookmarks as well.
Otherwise, the `setDatasetProperties` below fails with the error message we discovered in this issue:

https://github.com/zrepl/zrepl/pull/259#issuecomment-578481081

(zrepl uses bookmarks for resumable replication)

@stgraber could you fix this? I have no experience with lxd, just tried to root-cause the user issue in zrepl